### DISCUSSION TITLE
How to show a total at bottom of list ?

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#4929] [Help] [answered]
Hi All,

How to show a total at bottom of list ?
![image](https://user-images.githubusercontent.com/6214964/201534935-f8e93bb5-8b20-46ff-a090-348eb173c877.png)
If this  is possible then how I can archive that ?

Thank you

**Comment by Z3d0X** [comment, marked as answer]
As of now:
> You can override getTableContentFooter() on the Livewire Component and return a View object that contains your summary row HTML

In your case, the livewire component is the RelationManager

An alternative is to use a footer widget

[Upcoming Filament V3 supports a summary section on the table]( https://twitter.com/danjharrin/status/1591131923770150912?s=20&t=jFmv9vSymlsgpLluHPfY2Q)
